src/windows/views/webview.py

```diff
@@ -1890,6 +1890,20 @@ def Slice_Triggered(self, action, clip_ids, trans_ids, playhead_position=0):
         fps_float = fps_num / fps_den
         frame_duration = fps_den / fps_num
 
+        # Get CLIPS array from IDs
+        clips = get_app().project.get("clips")
+        clips = list(filter(lambda x: x.get("id") in clip_ids, clips))
+        # Get locked tracks from project
+        locked_layers = [
+            t.get("number") // 1000000
```

Comment:
Why are we dividing the layer # by 1000000? Let's remove this

Reply:
I figured since they're multiples of a million, the layer might be used to settle which clip or transition to draw over another.

... Speaking of which I'm gonna test this with transitions to make sure those are locked too

Reply:
Tried without the division having 3 clips on top of each other. They all had the same `layer`: 3000000.

So that's gone, and I made use of the loops that already exist.